FrameworkSystem/DB/UserProfileDB.py

```diff
@@ -9,6 +9,8 @@
 import sys
 import hashlib
 
+import cachetools
```

Comment:
From https://pypi.org/project/cachetools/: 
> Requires: Python ~=3.5

Reply:
It would also need to be added to the various requirements files here.

Reply:
It supported Python 2.7 prior to version 4 (we already use it in LHCbWebDIRAC).